docs/testing.rst

```diff
@@ -196,7 +196,6 @@ you can also read the `pytest markers`_ documentation for more options.
 Smoke tests
```

Comment:
Should we just remove this section, seeing as the smoke markers don't do anything any longer?

Reply:
This PR removes the machinery for running the smoke tests via local docker container, but it doesn't remove the smoke markers themselves, so the note in this section about running them from the command line still applies.

Reply:
As per my other comment, this PR now removes the smoke markers entirely, along with this section.